27/06/21 (08:46:39) | ERROR: Exception while getting cpu frequency!
27/06/21 (08:46:39) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/local/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
27/06/21 (08:46:59) | ERROR: Exception while getting cpu frequency!
27/06/21 (08:46:59) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/local/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
27/06/21 (09:01:03) | ERROR: Exception while getting cpu frequency!
27/06/21 (09:01:03) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/local/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
27/06/21 (15:37:00) | ERROR: Exception while getting cpu frequency!
27/06/21 (15:37:00) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/local/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
14/07/21 (15:35:50) | ERROR: Exception while getting cpu frequency!
14/07/21 (15:35:50) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/local/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
14/07/21 (21:28:39) | ERROR: Exception while getting cpu frequency!
14/07/21 (21:28:39) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/local/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
15/07/21 (20:16:41) | ERROR: Exception while getting cpu frequency!
15/07/21 (20:16:41) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/local/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
15/07/21 (20:17:33) | ERROR: Exception while getting cpu frequency!
15/07/21 (20:17:33) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/local/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
15/07/21 (20:22:30) | ERROR: Exception while getting cpu frequency!
15/07/21 (20:22:30) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/local/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
17/07/21 (11:52:01) | ERROR: Exception while getting cpu frequency!
17/07/21 (11:52:01) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/local/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
17/07/21 (11:58:34) | ERROR: Exception while getting cpu frequency!
17/07/21 (11:58:34) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/local/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
17/07/21 (11:58:56) | ERROR: Exception while getting cpu frequency!
17/07/21 (11:58:56) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/local/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
17/07/21 (12:11:38) | ERROR: Exception while getting cpu frequency!
17/07/21 (12:11:38) | ERROR: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
Traceback (most recent call last):
  File "/opt/local/bin/bpytop", line 3060, in _collect
    if CONFIG.show_cpu_freq and hasattr(psutil.cpu_freq(), "current"):
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/__init__.py", line 1859, in cpu_freq
    ret = _psplatform.cpu_freq()
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/psutil/_psosx.py", line 180, in cpu_freq
    curr, min_, max_ = cext.cpu_freq()
FileNotFoundError: [Errno 2] No such file or directory (originated from sysctlbyname('hw.cpufrequency'))
